ui/src/app/view/timestamp-editor/timestamp-editor.component.html

```diff
@@ -33,10 +33,17 @@ <h4>For Transport-Call {{ transportCall.transportCallSequenceNumber}}</h4>
 </div>
 
 <div class="p-fluid p-formgrid p-grid">
+
   <div class="p-field p-col-12 p-md-4">
     <label for="timestampType">{{ 'general.table.header.timestamp.label' | translate }}: </label>
-    <p-dropdown id="timestampType" styleClass="dropdown" [(ngModel)]="timestampSelected"
-                [options]="timestampTypes" (onChange)="showLocationNameOption()"></p-dropdown>
+    <p-dropdown id="timestampType" styleClass="dropdown" [(ngModel)]="timestampSelected" [options]="timestampTypes"
+      (onChange)="showLocationNameOption()"></p-dropdown>
+  
+    <div class="p-field p-col-12 p-md-2"></div>
+
+    <label for="terminalSelected">{{ 'general.table.header.timestamp.label' | translate }}: </label>
+    <p-dropdown id="terminalSelected" styleClass="dropdown" [(ngModel)]="terminalSelected" [options]="terminalOptions">
+    </p-dropdown>
```

Comment:
Terminal should not be shown for all timestamps (e.g. `XTA-PBP` never has terminal)

Reply:
This has been solved.